exercises/bank.js

```diff
@@ -1,38 +1,60 @@
 
 
 class Bank{
-	constructor(){
+	constructor(name){
 		//takes in the name of the bank.
 		//makes an object to store all accounts into
+		this.accounts = {};
 	}
-	makeAccount( ){
+	makeAccount( newAccount){
 		//makes an account
 		//takes in the account number of an account
 		//checks if the account number already exists in the account storage object in the constructor
 			//if already exists, returns false
 		//otherwise makes an new Account object, storing it into the storage object in the constructor
 		//returns the Account object that was constructed
+
+		if(this.accounts === newAccount){
+			return false;
+		} this.accounts[newAccount] = new Account();
+		console.log(this.accounts);
+			return this.accounts[newAccount];
 	}
-	checkForAccount(  ){
+
+	checkForAccount( accountNumber ){
 		//checks if an account exists or not
+		if(this.accounts.hasOwnProperty(accountNumber)){
+			return true;
+		} return false;
 		//takes in the account number of an account
 		//returns false if the account does not exist in the accounts object from the constructor
 		//returns true if the account does exist
 	}
-	removeAccount(  ){
+	removeAccount( accountNumber ){
 		//removes an account
+		if(!this.accounts.hasOwnProperty(accountNumber)){
+			return "account "+accountNumber+" does not exist";
+		} else if (this.accounts[accountNumber].balance > 0){
```

Comment:
you should not directly access the property of the object, you should access the getAmount method of the object